ArcGISRuntimeSDKQt_CppSamples/Geometry/FormatCoordinates/FormatCoordinates.cpp

```diff
@@ -0,0 +1,163 @@
+// [WriteFile Name=DisplayMap, Category=Geometry]
+// [Legal]
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "FormatCoordinates.h"
+
+#include "Basemap.h"
+#include "CoordinateFormatter.h"
+#include "Graphic.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "Point.h"
+#include "SimpleMarkerSymbol.h"
+
+#include <QDebug>
+
+using namespace Esri::ArcGISRuntime;
+
+FormatCoordinates::FormatCoordinates(QQuickItem* parent) :
+    QQuickItem(parent)
```

Comment:
please update indenting to be 2 spaces for all C++ files. QML should remain as 4